src/lib/components/colors/Formats.svelte

```diff
@@ -30,18 +30,18 @@
 </script>
 
 <div data-content="formats" data-column="tool">
-  <h4 class="small-only label">{displayType} Color</h4>
+  <h2 class="small-only label">{displayType} Color</h2>
```

Comment:
This only displays at small sizes, and I think it makes sense as an h2 (especially since the other "background Color" is a label and not a heading) but it is a bit weird then to have 2-3 groups of colors formats at the same heading level underneath when they should likely be `<h3>` but this `<h2>` doesn't display at larger screen sizes so Im not sure whats the better way to handle this? Maybe this isn't a heading at all and just a span with a class that makes it look like a heading?

Reply:
I agree this is a strange situation – mainly caused by the back-and-forth background/foregroud/background/foreground source order. I wonder if we should rethink it at that level? Like what if we collapsed the format lists on mobile into a details/summary that stays connected to the correct field? Then these headings aren't needed.

Reply:
Made a new issue to take a look at this: https://github.com/oddbird/oddcontrast/issues/156